source/scripts/common/chains.ts

```diff
@@ -1,5 +1,10 @@
 import { AssetType } from "state/vault/types";
 
+enum StargazerProvider {
+  CONSTELLATION = 'constellation',
+  ETHEREUM = 'ethereum',
+}
+
 enum StargazerChain {
   CONSTELLATION = 'constellation',
   ETHEREUM = 'ethereum',
```

Comment:
I just followed the references of `StargazerChain` and `StargazerProvider`, I may think you need to cleanup/reorganize uses for both as I think you're mixing things up with those.